framework/packages/abstract-interface/src/native/version_control.rs

```diff
@@ -139,22 +139,30 @@ impl<Chain: CwEnv> VersionControl<Chain> {
 
     /// Approve any abstract-namespaced pending modules.
     pub fn approve_any_abstract_modules(&self) -> Result<(), crate::AbstractInterfaceError> {
-        let proposed_abstract_modules = self.module_list(
+        self.approve_any_namespace_modules(Namespace::unchecked(ABSTRACT_NAMESPACE))
+    }
+
+    /// Approve any "namespace" pending modules.
+    pub fn approve_any_namespace_modules(
```

Comment:
Would like to rename this as this suggests we approve modules for any namespace. 
```suggestion
    pub fn approve_any_module_for_namespace(
```

Reply:
updated!